### PR TITLE
Fix no-tls1_3

### PR DESCRIPTION
**Pull request by mattcaswell**
Also fixes no-tls1_2

### CODE REVIEW

**Comment by mattcaswell** [comment]
Fixup pushed to address a CI failure.

File: test/sslapitest.c [hunk 1]
```diff
@@ -7889,6 +7890,15 @@ static int test_sigalgs_available(int idx)
                                        &sctx, &cctx, cert, privkey)))
         goto end;
 
+    /* Ensure we only use TLSv1.2 ciphersuites based on SHA256 */
+    if (idx < 4) {
+    if (!TEST_true(SSL_CTX_set_cipher_list(cctx, "ECDHE-RSA-AES128-GCM-SHA256")))
+        goto end;
+    } else {
+    if (!TEST_true(SSL_CTX_set_cipher_list(cctx, "ECDHE-ECDSA-AES128-GCM-SHA256")))
+        goto end;
```
Comment (t8m) [inline test/sslapitest.c:7899]: Wrong formatting.

**Comment by mattcaswell** [comment]
Fixup pushed addressing feedback. Please take another look.

**Review by t8m — APPROVED** [review]

**Comment by mattcaswell** [comment]
Pushed. Thanks.

**Pull request closed without merging**